index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact">
+        <div class="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+          <input type="text" name="name" class="input" placeholder="Full name">
+          <input type="email" name="email" class="input" placeholder="Email address">
+          <textarea class="textarea" name="message" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] We can also add the `required` attribute to all of our inputs, this will ensure that our users fill all of the fields from our form before submitting it.